docs/UserGuide.md

```diff
@@ -82,49 +96,48 @@ Some examples of **invalid** user inputs for the *add command* are provided belo
 
 ## Features
 
-
-
 ### Viewing help : `help`
 
-Provides a link to the full User Guide that users can visit by clicking on the button 'Open in  Browser'.
-
-![help message](images/helpMessage.png)
+Provides a reference for commands in StudyTracker.
+You can also visit the User Guide and Developer Guide by clicking on the buttons in the window.
+![Help window](images/helpMessage.png)
 
 Format: `help`
 
-
 ### Adding a study spot: `add`
 
 Adds a study spot to the StudyTracker.
 
-Format: `add n/NAME* r/RATING* a/ADDRESS t/TAG... m/AMENITY...`
+**Format:** `add n/NAME* r/RATING* a/ADDRESS t/TAG... m/AMENITY...`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 Tags, amenities and address are all optional.
 </div>
 
-Examples:
+**Examples:**
 ```
 add n/COM1 Basement r/5
 New study spot “COM1 Basement” added to list!
```

Comment:
Actual output: 
`New study spot added: COM1 Basement; Rating: 5; Operating Hours: -; Address: -; Favourite: false`

It's a little different from the expected output above